frost-core/src/frost/keys/repairable.rs

```diff
@@ -0,0 +1,133 @@
+//! Repairable Threshold Scheme
+//!
+//! Implements the Repairable Threshold Scheme (RTS) from [https://eprint.iacr.org/2017/1155].
```

Comment:
I gave the wrong suggestion early, sorry :grimacing: 
To make Rust doc happy I'll go ahead and commit this after commenting

```suggestion
//! Implements the Repairable Threshold Scheme (RTS) from <https://eprint.iacr.org/2017/1155>.
```